proto/desmos/profiles/legacy/v200/models_app_links.proto

```diff
@@ -0,0 +1,155 @@
+syntax = "proto3";
+package desmos.profiles.legacy.v200;
+
+option go_package = "github.com/desmos-labs/desmos/v2/x/profiles/legacy/v200";
```

Comment:
Why v200 but not v230? I think the legacy version should be the last version.

Reply:
because it was since v2.0.0 that the `appLinks` had not been changed. So I thought they should belong to that particular version. Should we move this to v2.3.0?

Reply:
You are right, I thought it means the legacy `appLink` is available until v2.3.0.  Then v2.3.0 chain link should be v2.1.0...

Reply:
@dadamu actually also your thought makes sense. @RiccardoM How should we handle legacy code?

Reply:
I'm actually looking into this problem inside #730 by reviewing where different things are located for legacy stuff and updates